README.md

```diff
@@ -45,3 +45,25 @@ log as requests are made to them.
 You can test out the FIDS/Alerts sample application by visiting http://localhost:8080 in
 your web browser (if not running Docker locally, use the Docker host's
 address).
+
+## Alerts Backend Note:
+
+Please note that when you create an alert, whenever an event is triggered for that
+alert, AeroAPI will send a POST request to your configured endpoint. In order to configure
+your endpoint, the server you are running the app on will need to have a publicly exposed
+address/port (using the POST_PORT environment variable, NOT the WEB_SERVER_PORT variable).
+Furthermore, as noted in the docker-compose-alerts.yml file, we encourage the service for
+accepting POSTed triggered alerts to be isolated, and thus will be sent to a different Docker
+container. This means that you will have to set the endpoint URL using /post, instead of /api/post.
+You can then use this address in setting the endpoint by sending a PUT request to AeroAPI
+(see the [documentation](https://flightaware.com/aeroapi/portal/documentation#put-/alerts/endpoint)
+on how to set it). NOTE: if you already had a previously configured endpoint URL for other alerts,
+**you will change the same URL!** This means that ALL of your configured alerts, including
+ones configured on FlightAware's website, will all be sent to the newly configured endpoint.
+You can see this newly configured URL by going to FlightAware's
+[push notification testing page](https://flightaware.com/commercial/aeroapi/send.rvt)
+or by seeing it on the alert creation page on the webapp. On this push notification testing
+page, you can also do two things: test your configured endpoint on the backend to see if it
```

Comment:
Let's start a new paragraph detailing the use of the notification testing page, and break this up into a few sentences